src/core/lib/iomgr/tcp_posix.cc

```diff
@@ -424,74 +438,145 @@ static void call_read_cb(grpc_tcp* tcp, grpc_error* error) {
   GRPC_CLOSURE_SCHED(cb, error);
 }
 
-#define MAX_READ_IOVEC 4
+#define MAX_READ_IOVEC 128
```

Comment:
this too. is there a rationale for this or is it one of those commonly accepted values?

Reply:
This is a common value for TCP reads, but it's not necessary here. In an earlier version of this commit, I changed the allocation to be multiple of specific page sizes. If/when we do that we should up this from 4 to 128. I'd revert this one, because at the moment we will have at most 2 IOVs on read.

Reply:
no need to revert, but a comment will be nice

Reply:
I reverted because it would unnecessarily increase the stack size, and we really don't need it. In a follow up when I come up with a proper way of using fixed-size blocks, I will update this with a comment. Thanks.